crates/sovereign-sdk/rollup-interface/Cargo.toml

```diff
@@ -43,14 +44,15 @@ fuzzing = ["proptest", "proptest-derive", "sha2", "std"]
 testing = ["native"]
 arbitrary = ["dep:arbitrary"]
 std = [
-    "anyhow/default",
-    "borsh/default",
-    "borsh/bytes",
-    "bytes",
-    "digest/default",
-    "hex/default",
-    "proptest?/default",
-    "serde/default",
-    "sha2?/default",
-    "thiserror"
+  "anyhow/default",
+  "borsh/default",
+  "borsh/bytes",
+  "bytes",
+  "digest/default",
+  "hex/default",
+  "proptest?/default",
+  "serde/default",
+  "sha2?/default",
+  "thiserror",
+  "tracing",
```

Comment:
only put in native and it'll be solved

Reply:
I mean tracing

Reply:
i did

Reply:
Ahh i see what you mean... it's in std so it should also be included with native

Reply:
no its in std which is in native. I believe we should put it explicitly in native, like tokio